docs/mms_on_fargate.md

```diff
@@ -0,0 +1,256 @@
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/container+health+checks.png)
+
+and the environment, with the entry point that we have discussed earlier:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/entrypoint.png)
+
+Everything else can be left as default. So feel free to press add and finish the creation of our very first task. If everything is ok you should now be able to see your task in the list of task definitions.
+
+Finally we have a task that we can be executed, but we need something that will be in charge of the actual execution. This something, in terms of ECS, is called Service. Service is capable of running multiple tasks and in charge of making sure that required amount of tasks is always running, restarting un-health tasks adding more tasks when needed, etc. If the service is going to be accessible from the Internet, we first need to configure load balancer that will  be in charge of serving the traffic from the Internet and redirecting it to the tasks. So we are going to create an LB now:
+
+## Create LoadBalancer
+
+AWS Supports several different types of Load Blanacers:
+
+
+* Application Load Balancer, that works on the level 7 of the OSI model (effectively with the HTTP/HTTPS protocols)
+* TCP Load Balancer and
+* deprecated Elastic Load Balancer, which also used to work on the TCP level
+
+For our cluster we are going to use application load balancer.
+1. Open AWS console
+2. Goto EC2
+3. Goto to “Load balancers” section
+4. Create new Load Balancer
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/1__Create_Load_Balancer.png)
+
+5. Choose “HTTP/HTTPS”
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/2__HTTP_HTTPS+.png)
+
+6. Set all the required details. Most importantly set the VPC. VPC is very important. Having wrong VPC might cause our LB to not be able to communicate with your tasks. So memorize VPC that you going to use here for later.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3_2_Listeners_and_AZ+.png)
+
+7. Next is security group. This is also important. Your security group should:
+
+* allow inbound connection for port 80 (since this is the port on which LB will be listening on)
+* member of the security group should be able to talk to the security group where you planing to have your service. In our case we will use same security group everywhere in order to simplify our life.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/4.+Configure+Security+groups.png)
+
+8. Routing configuration is simple. The thing is here you need to create what is called “target group”. But the thing is, ECS will create required service group later on for you. Therefore now you need to create dummy “target group” that you will delete after the creation of the LB. Yes, I know, there should be option to skip this, but there is not :(
```

Comment:
simple. (But it isn't... it's making you jump through hoops. I'd cover the "ok this step is a bit convoluted" part up front, and delete the last line of this paragraph.)

You need to...
Delete "But the thing is,"

the required

Therefore,

a dummy

Reply:
Done